test/runAnalysis/runAnalysis.py

```diff
@@ -366,7 +366,7 @@ def main():
 					cmssw_tmp = cmssw_tmp.replace('mc_2017',dataset_name)
 
                                 # write add filename information
-				if args.add2016:
+				if True: #args.add2016:
```

Comment:
We had this discussion before: Doing it like this breaks existing functionality that we need! Please do not do it this. You're way to run locally over full samples is nice, but please add it as a new option and don't use -s for it ;)

Reply:
Sure no problem! ;)